script/build-bisect.py

```diff
@@ -85,6 +85,8 @@ def get_releases(repo):
             if release['draft']:
                 draft_count = draft_count + 1
                 continue
+            if 'android' in release['tag_name'] or 'b' in release['tag_name']:
```

Comment:
Also, this could be made more general (I just got it to work with these alphabetic characters to make it function for our testing tonight), with the following:

```
if not release['tag_name'].replace('.','').isdigit():
```

Reply:
Good idea! 😄